image_generation/lcm_dreamshaper_v7/cpp/README.md

```diff
@@ -0,0 +1,103 @@
+* [OpenVINO](https://docs.openvino.ai/2023.2/openvino_docs_install_guides_overview.html): Model inference
+* Eigen3
+
+Prepare a python environment and install dependencies:
+```shell
+conda create -n openvino_lcm_cpp python==3.10
+conda activate openvino_lcm_cpp
+conda install openvino eigen c-compiler cxx-compiler make
+```
+
+## Step 2: Latent Consistency Model and Tokenizer models
+
+### Latent Consistency Model model
+
+1. Install dependencies to import models from HuggingFace:
+
+    ```shell
+    conda activate openvino_lcm_cpp
+    python -m pip install -r scripts/requirements.txt
+    python -m pip install ../../../thirdparty/openvino_contrib/modules/custom_operations/[transformers]
+    ```
+
+2. Run model conversion script to download and convert PyTorch model to OpenVINO IR via [optimum-intel](https://github.com/huggingface/optimum-intel). Please, use the script `scripts/convert_model.py` to convert the model:
+
+    ```shell
+    cd scripts
+    python convert_model.py -lcm "SimianLuo/LCM_Dreamshaper_v7" -t FP16
+    ```
+
+> [!NOTE]
+>Only static model is currently supported for this sample.
+
+### LoRA enabling with safetensors
+
+Refer to [python pipeline blog](https://blog.openvino.ai/blog-posts/enable-lora-weights-with-stable-diffusion-controlnet-pipeline).
+The safetensor model is loaded via [safetensors.h](https://github.com/hsnyder/safetensors.h). The layer name and weight are modified with `Eigen Lib` and inserted into the LCM model with `ov::pass::MatcherPass` in the file [common/diffusers/src/lora.cpp](https://github.com/openvinotoolkit/openvino.genai/blob/master/image_generation/common/diffusers/src/lora.cpp).
+
+LCM model [lcm_dreamshaper_v7](https://huggingface.co/SimianLuo/LCM_Dreamshaper_v7) and Lora [soulcard](https://civitai.com/models/67927?modelVersionId=72591) are tested in this pipeline.
+
+Download and put safetensors and model IR into the models folder.
+
+## Step 3: Build the LCM application
+
+```shell
+conda activate LCM-CPP
+cmake -DCMAKE_BUILD_TYPE=Release -S . -B build
+cmake --build build --config Release --parallel
+```
+
+## Step 4: Run Pipeline
+```shell
+./build/lcm_dreamshaper [-p <posPrompt>] [-s <seed>] [--height <output image>] [--width <output image>] [-d <device>] [-r <readNPLatent>] [-a <alpha>] [-h <help>] [-m <modelPath>] [-t <modelType>]
+
+Usage:
+  lcm_dreamshaper [OPTION...]
+```
+
+* `-p, --posPrompt arg` Initial positive prompt for SD  (default: cyberpunk cityscape like Tokyo New York  with tall buildings at dusk golden hour cinematic lighting)
+* `-d, --device arg`    AUTO, CPU, or GPU (default: CPU)
+* `--step arg`          Number of diffusion step ( default: 20)
+* `-s, --seed arg`      Number of random seed to generate latent (default: 42)
+* `--num arg`           Number of image output(default: 1)
+* `--height arg`        Height of output image (default: 512)
+* `--width arg`         Width of output image (default: 512)
```

Comment:
it's stated that `Only static model is currently supported for this sample`. So, do we need `height` and `width` parameters then?